app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+import StaffCard from './StaffCard'
+
+const { Flex, Box } = require('grid-styled')
+
+const Grid = styled(Flex)`
+  flex-flow: row wrap;
+`
+const Card = styled(Box)`
+  padding-top: 20px;
```

Comment:
If we go with 24px instead of 20px, then we can use a theme variable `${th('space.3')}`.

Is there any particular reason that it's 20px right now?